Documentation/CommandReference/Index.rst

```diff
@@ -636,6 +642,9 @@ Directories of inactive extension are **removed** from ``typo3/sysext`` and ``ty
 This is a one way command with no way back. Don't blame anybody if this command destroys your data.
 **Handle with care!**
 
+This command is deprecated.
```

Comment:
What about `typo3/sysext`? How would you get rid of them without this command? Or is the deprecation aiming at TYPO3 v9 where we (hopefully) finally have separate `typo3/*` packages?

Reply:
actually subtree splits will be available for TYPO3 8 as well. We only need to figure out whether something will break when we publish them on packagist.

besides that I have a package ready, which creates typo3/sysext as folder and links individual core packages into this folder. It only needs to be documented. benefit is: you can activate any new core extension, without having to re-download in case you deleted some exts already.

It only needs to be documented and communicated ;)

did you actually use this command? I never used it much

Reply:
No, I never used it but always planned to do so. Of course, if I can simply pull in what I need instead of dropping what I don't need I'm perfectly fine.